cmake/modules/OmrDDRSupport.cmake

```diff
@@ -153,6 +152,11 @@ function(target_enable_ddr tgt)
 		# if no value is present, assume the debug info lives in the target file
 		if(NOT debug_file)
 			set(debug_file "$<TARGET_FILE:${tgt}>")
+		else()
+			# For non flattened debug on osx, the debug "file" is actually a folder
+			if(OMR_OS_OSX AND NOT OMR_FLATTEN_DEBUG_INFO)
+				set(debug_file "${debug_file}/Contents/Resources/DWARF")
```

Comment:
This is a directory: the file we want is:
```
set(debug_file "${debug_file}/Contents/Resources/DWARF/$<TARGET_FILE_NAME:${tgt}>")
```
For example, for `libomrsig.dylib`
it should be `libomrsig.dylib.dSYM/Contents/Resources/DWARF/libomrsig.dylib`.